### DISCUSSION TITLE
Free download manager give hash mishmatch

### DISCUSSION THREAD

**Discussion by GiulianoSpaghetti** [microsoft/winget-pkgs#112688] [Q&A]
Hello,
winget install softdeluxe.freedownloadmanager gives hash mismatch.

<img width="867" alt="2023-07-20" src="https://github.com/microsoft/winget-pkgs/assets/49764967/e51e8b76-215a-4bcf-b621-3b33668e8921">

**Comment by rodrigoborgesdeoliveira** [comment]
Hi @numerunix, thanks for reporting this. I've opened an issue (#112850) to track this. If you find any other problems with this or another package, please open an issue [here](https://github.com/microsoft/winget-pkgs/issues/new/choose) if there isn't one already.
